ph5/utilities/ph5validate.py

```diff
@@ -331,30 +331,33 @@ def check_response_t(self, resp_check_info):
             "check will be ceased to process.")
         header = ("-=-=-=-=-=-=-=-=-\n"
                   "Response_t\n"
-                  "%s error, 0 warning, 0 info\n"
+                  "%s error, %s warning, 0 info\n"
                   "-=-=-=-=-=-=-=-=-\n")
         checked_data_files = {}
-        unique_filenames_n_i = []
         self.ph5.read_response_t()
         errors = set()
-        if not validation.check_resp_load(
-                self.ph5.Response_t, errors, LOGGER):
-            header %= len(errors)
-            return [ValidationBlock(heading=header, error=errors)]
+        if validation.check_has_response_filename(
```

Comment:
Are validation.check_has_response_filename and validation.check_resp_load equivalent?

Reply:
Holly recommended that the error message should include metadatatoph5(), not just resp_load(). So the name check_resp_load didn't cover that and was misleading. So I changed name to check_has_response_filename to be more descriptive.

Reply:
This make sense. Thanks for pointing this out.